Docker build fails with error[E0505]

During the build of the docker image, the following error appears.  I googled the error extensively, but did not find any reference to it.


```  
...
Compiling stderrlog v0.4.1
Compiling rayon v1.2.0
Compiling cc v1.0.26
Compiling configure_me v0.3.3
error[E0505]: cannot move out of `string` because it is borrowed
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/configure_me_codegen-0.3.9/src/config.rs:58:25
   |
55 |             for (i, c) in string.chars().enumerate() {
   |                           ------ borrow of `string` occurs here
...
58 |                         string,
   |                         ^^^^^^ move out of `string` occurs here

error: aborting due to previous error

For more information about this error, try `rustc --explain E0505`.
error: Could not compile `configure_me_codegen`.
warning: build failed, waiting for other jobs to finish...
error: build failed
The command '/bin/sh -c cargo build --release' returned a non-zero code: 101
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -12,6 +12,9 @@ readme = "README.md"
 edition = "2018"
 build = "build.rs"
 
+[package.metadata.configure_me]
+spec = "config_spec.toml"
+
 [profile.release]
 lto = true
 
@@ -47,4 +50,4 @@ time = "0.1"
 tiny_http = "0.6"
 
 [build-dependencies]
-configure_me_codegen = "0.3.8"
+configure_me_codegen = "0.3.12"

--- a/build.rs
+++ b/build.rs
@@ -1,5 +1,5 @@
 extern crate configure_me_codegen;
 
 fn main() -> Result<(), configure_me_codegen::Error> {
-    configure_me_codegen::build_script_with_man("config_spec.toml")
+    configure_me_codegen::build_script_auto()
 }

